Index: b/bin/check_lsb-services.sh
===================================================================
--- a/bin/check_lsb-services.sh	2023-04-19 15:51:21.665289659 -0400
+++ b/bin/check_lsb-services.sh	2023-04-19 15:51:22.510712087 -0400
@@ -39,7 +39,7 @@
 res=`expr match "$stat" '.*\(not running\.*\)'`
 
 if [[ $res = "" ]]; then
-  res=`expr match "$stat" '.*\( running\.*\)'`
+  res=`expr match "$stat" '.*\( *running\.*\)'`
   if [[ $res = "" ]]; then
     exit $EXIT_CRITICAL
   else
