--- a/bin/check_lsb-services.sh	2022-11-28 16:35:39.722291748 +0000
+++ b/bin/check_lsb-services.sh	2023-03-28 15:51:39.473941180 +0000
@@ -39,7 +39,7 @@
 res=`expr match "$stat" '.*\(not running\.*\)'`
 
 if [[ $res = "" ]]; then
-  res=`expr match "$stat" '.*\( running\.*\)'`
+  res=`expr match "$stat" '.*\( *running\.*\)'`
   if [[ $res = "" ]]; then
     exit $EXIT_CRITICAL
   else
--- a/bin/check_lsb-services.sh	2023-04-10 09:22:50.187282248 -0400
+++ b/bin/check_lsb-services.sh	2023-04-10 09:22:50.643626390 -0400
@@ -39,7 +39,7 @@
 res=`expr match "$stat" '.*\(not running\.*\)'`
 
 if [[ $res = "" ]]; then
-  res=`expr match "$stat" '.*\( running\.*\)'`
+  res=`expr match "$stat" '.*\( *running\.*\)'`
   if [[ $res = "" ]]; then
     exit $EXIT_CRITICAL
   else
